quantum/rgblight.c

```diff
@@ -116,7 +116,7 @@ void eeconfig_update_rgblight(uint32_t val) {
   eeprom_update_dword(EECONFIG_RGBLIGHT, val);
 }
 void eeconfig_update_rgblight_default(void) {
-  dprintf("eeconfig_update_rgblight_default\n");
```

Comment:
Why are we removing all of these prints?

Reply:
I put them back, that was something for debugging I forgot to undo